libraries/grpc-sdk/src/helpers/RoutingUtilities.ts

```diff
@@ -22,6 +22,8 @@ message RouterRequest {
 message RouterResponse {
   string result = 1;
   string redirect = 2;
+  optional string setCookies = 3;
```

Comment:
We [don't](https://github.com/ConduitPlatform/Conduit/issues/86) currently follow this yet, but gRPC's style guide suggests that we [use underscore_separated_names for message field names](https://developers.google.com/protocol-buffers/docs/style#message_and_field_names).